### PR TITLE
Wagtail data

### PR DESCRIPTION
**Pull request by cadecairos**
Related issue: #1308
Summary: Updates the load_fake_data script to generate Wagtail pages and other content. Also adds a post deploy step for review apps that adds a super user and logs the password to the build logs.

- [x] Build pages
- [x] Remove crufty generate steps
- [x] update readme
- [x] update build steps 
- [x] Build homepage highlights

To test:

1. Run from a freshly migrated DB: `inv manage load_fake_data`
2. Run a second time: `inv manage load_fake_data`
3. Run with the delete Flag: `inv manage load_fake_data -f delete`

After each step, check the site, and the CMS for issues (pages load, no duplicate homepages/sites, etc)

Also, you can run the code that creates a super user with `inv manage review_app_admin`

### CODE REVIEW

**Comment by patjouk** [comment]
screen shot 2018-05-09 at 3 49 34 pm

I have a strange top double menu: I tried fake data with and without the delete flag.

I'll continue to look around but 🥇 already for this awesome work!

File: Pipfile [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
+"boto3" = "*"
+python-slugify = "*"
+Pillow = "*"
+django-cors-headers = "*"
+django_csp = "*"
+django-admin-sortable = "*"
+requests = "*"
+factory_boy = "*"
+Faker = "*"
+wagtail-factories = {git = "https://github.com/mvantellingen/wagtail-factories@1ead51cadaad3b4530ba2197ccf45d2dca87dbdf", editable=true}
```
[the remark this answers is not in the document]
Comment (cadecairos) [reply Pipfile]: This was master, at the time of writing. The latest release used an old version of factory_boy, which broke all the existing factory code. We can check to see if they tagged a new release.

File: Pipfile [hunk 2]
```diff
@@ -1,38 +1,39 @@
...
-
-[requires]
-python_version = "3.6"
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+Django = "==1.11.12"
```
[the remark this answers is not in the document]
Comment (cadecairos) [reply Pipfile]: 👍

File: network-api/networkapi/management/commands/flush_models.py [hunk 3]
```diff
@@ -21,21 +20,6 @@ def handle(self, *args, **options):
...
-        self.stdout.write('Dropping LandingPage objects...')
-        LandingPage.objects.all().delete()
-
-        self.stdout.write('Dropping Signup objects...')
-        Signup.objects.all().delete()
-
-        self.stdout.write('Dropping CampaignPage objects...')
-        Campaign.objects.all().delete()
-
-        self.stdout.write('Dropping Highlight objects...')
```
Comment (gideonthomas) [inline network-api/networkapi/management/commands/flush_models.py]: don't we still need `Highlight` object cleanup? We create them in `load_fake_data`

Comment (cadecairos) [reply network-api/networkapi/management/commands/flush_models.py]: yup, good catch

**Comment by cadecairos** [comment]
@gideonthomas updated the PR to fix the uncaught exceptions around the statically named pages. I've also added the Highlights model back into the flush_models script.

Another thing you can test - is the output reproducible when passing in a seed value?
`pipenv run python network-api/manage.py load_fake_data --delete --seed testseedvalue` <- run that twice, check homepage/page titles/people to see if they're the same.

**Pull request merged**